### PR TITLE
add welsh support to reporting for COF

### PR DESCRIPTION
**Pull request by Adam-W1**
https://digital.dclg.gov.uk/jira/browse/FS-2464

### Change description
Include the key application report metrics for welsh applications

- [x] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
usual 'pytest' command will cover these changes as test_reports.py

### CODE REVIEW

File: db/queries/reporting/queries.py [hunk 1]
```diff
@@ -102,19 +109,32 @@ def get_report_for_applications(
...
-                for question in form["questions"]:
+        if application["language"] == "en":
+            report_config_forms = [
+                report_config.get("form_name")
+                for report_config in KEY_REPORT_MAPPING
+            ]
+        elif application["language"] == "cy":
+            report_config_forms = [
+                report_config.get("form_name_cy")
+                for report_config in KEY_REPORT_MAPPING
```
Comment (gidsg) [inline db/queries/reporting/queries.py:120]: Could we do this in one assignment/statement e.g.
```
report_config.get("form_name_cy") if application["language"] == "cy" else report_config.get("form_name")
```
(untested but hopefully gives an idea)

Comment (Adam-W1) [reply db/queries/reporting/queries.py:120]: This worked thanks @gidsg 
`        report_config_forms = [report_config.get("form_name_cy") if application['language'] == 'cy' else report_config.get("form_name") for report_config in KEY_REPORT_MAPPING]
`

File: db/queries/reporting/queries.py [hunk 2]
```diff
@@ -35,36 +35,43 @@ def get_general_status_applications_report(
...
         "form_name": "asset-information",
+        "form_name_cy": "gwybodaeth-am-yr-ased",
         "key": "yaQoxU",
         "return_field": "asset_type",
     },
     {
         "form_name": "project-information",
+        "form_name_cy": "gwybodaeth-am-y-prosiect",
         "key": "yEmHpp",
         "return_field": "geography",
```
Comment (rayhan15243) [inline db/queries/reporting/queries.py:64]: @Adam-W1 Why is the return_field "geography"? I would have assumed it returned the project name field?

File: tests/test_reports.py [hunk 3]
```diff
@@ -167,10 +187,17 @@ def test_get_applications_report_query_param(
...
-    for line in line2, line3:
-        field1, field2, _, _, field5, _, _ = line.split(",")
-        # could also do this to ignore all fields after 5.
-        # field1, field2, _, _, field5, *_ = line.split(",")
-        assert field1 == "Test Reference Number"
-        assert field2.startswith("Test Org Name ")
-        assert field5 == "W1A 1AA"
+
+    field1, field2, _, _, field5, _, _ = line2.split(",")
+    # could also do this to ignore all fields after 5.
```
Comment (srh-sloan) [inline tests/test_reports.py:192]: I think this got committed as part of discussion on my previous PR so can be removed :) Here and line 199

**Review by srh-sloan — COMMENTED** [review]
Tiny comment, otherwise looks good! Nice use of parametrization in the tests with language 👍

File: db/queries/reporting/queries.py (hunk 2 above)
Comment (Adam-W1) [reply db/queries/reporting/queries.py:64]: unsure tbh @rayhan15243  I didnt come up with this naming convention - it does return the UK address but maybe the rrequriement was that this is called "geography" in the report

**Review by rayhan15243 — APPROVED** [review]

File: db/queries/reporting/queries.py (hunk 2 above)
Comment (rayhan15243) [reply db/queries/reporting/queries.py:64]: @Adam-W1 Ah okay cool. I've approved

**Pull request merged**